src/main/java/Baymax.java

```diff
@@ -0,0 +1,163 @@
+                String[] timeParts = parts[1].split(" /to ");
+                baymax.addEvent(parts[0], timeParts[0], timeParts[1]);
+                continue;
+            }
+            //delete Task
+            if (userInput.startsWith("delete")) {
+                int taskNumber = Integer.parseInt(userInput.split(" ")[1]);
+                baymax.delete(taskNumber);
+                continue;
+            }
+            //Error 3: handled invalid commands
+            System.out.println("Invalid command.");
+        }
+        //close scanner
+        scanner.close();
+    }
+    //add Todo to taskList
+    private void addTodo(String task) {
+        Todo newTask = new Todo(task);
+        taskList.add(newTask);
+        Storage.saveTasks(taskList);
+        System.out.println("______________________________________________________________\n"
+                + "added: " + newTask + "\n"
+                + "______________________________________________________________\n");
+    }
+    //add Deadline to taskList
+    private void addDeadline(String task, String by) {
+        Deadline newTask = new Deadline(task, by);
+        taskList.add(newTask);
+        Storage.saveTasks(taskList);
+        System.out.println("______________________________________________________________\n"
+                + "added: " + newTask + "\n"
+                + "______________________________________________________________\n");
+    }
+    //add Event to taskList
+    private void addEvent(String task, String from, String to) {
+        Event newTask = new Event(task, from, to);
+        taskList.add(newTask);
+        Storage.saveTasks(taskList);
+        System.out.println("______________________________________________________________\n"
+                + "added: " + newTask + "\n"
+                + "______________________________________________________________\n");
+    }
+
+    private void printList() {
+        System.out.println("______________________________________________________________");
+        for (int i = 0; i < taskList.size(); i++) {
+            int taskNumber = i + 1;
+            System.out.println(taskNumber + ". " + taskList.get(i));
+        }
+        System.out.println("______________________________________________________________");
+    }
+
+    private void markAsDone(int taskNumber) {
+        taskList.get(taskNumber - 1).markAsCompleted();
+        Storage.saveTasks(taskList);
+        System.out.println("______________________________________________________________");
+        System.out.println("Good job on completing the task!\n"
+                        + taskList.get(taskNumber - 1));
+        System.out.println("______________________________________________________________");
+    }
+
+    private void unmark(int taskNumber) {
+        //Error2: Handled invalid unmark function
+        if (!taskList.get(taskNumber - 1).getIsCompleted()) {
```

Comment:
I think `isCompleted` should suffice
```suggestion
        if (!taskList.get(taskNumber - 1).isCompleted()) {
```